### DISCUSSION TITLE
NGAP is not staying  up when srsRAN and open5gs deployed in 2 different hosts

### DISCUSSION THREAD

**Discussion by gakest2000** [srsran/srsRAN_Project#184] [General Help] [answered]
I have deployed srsRAN and open5gs in two different hosts.
set up correctly the NGAP and GTPU addresses on open5gs and srsRAN. and aligned PLMN and TAC
When starting the services, the gNodeb is accepted and after few seconds disconnected without any reason
the logs are not showing the reason, same as wireshark
What can be the cause? does srsRAN support open5GS in another machine?
but when deploying on the same host, the NGAP is stable up.

**Comment by pgawlowicz** [comment, marked as answer]
Please try this patch and provide a feedback if it works
https://github.com/srsran/srsRAN_Project/issues/172#issuecomment-1681908406

**Reply by gakest2000** [reply]
Thanks dear, it worked for me, but I needed to rebuild srsRAN after applied the patch.
Thank you
